king-local.json

```diff
@@ -0,0 +1,3 @@
+{
+  "king.gds_file": "gs://topmed_workflow_testing/UWGAC_WDL/ALL.pruned.gds"
```

Comment:
Since this local JSON the exact same as the terra one, I don't think it's really needed. I only create local JSONs when I'm testing with big files I don't want to keep downloading from Google, so I just have them link to my filesystem instead.

Reply:
Do you suggest I just delete this particular json then?